apps/dh/api-dh/source/DataHub.WebApi/DataHub.WebApi.csproj

```diff
@@ -41,7 +41,7 @@ limitations under the License.
     <PackageReference Include="Energinet.DataHub.Charges.Clients.Registration" Version="6.0.0" />
```

Comment:
Update to our new middleware and let internal_openid_url be null or empty string.